digitalearthau/coherence.py

```diff
@@ -111,24 +166,24 @@ def _archive_duplicate_siblings(dc, ids):
     return len(older_duplicates)
 
 
-def _check_ancestors(check_ancestors: bool,
-                     check_siblings: bool,
+def _check_ancestors(check_siblings: bool,
                      archive_siblings: bool,
                      dc: Datacube,
                      dataset: Dataset):
-    global _siblings_count
-    ancestors_archive_count = 0
+    global _siblings_count, _archive_count
+
     dataset = dc.index.datasets.get(dataset.id, include_sources=True)
     if dataset.sources:
         for classifier, source_dataset in dataset.sources.items():
             if source_dataset.is_archived:
                 _LOG.info(
-                    "ancestor.dataset_id",
+                    "ancestor." + str(dataset.type.name) + ".dry_run",
```

Comment:
Feel free to use the new Python 3.6 style f-strings in `digitalearthau`.
This can be simplified to `f"ancestor.{dataset.type.name}.dry_run"`